build/android/BUILD.gn

```diff
@@ -5,22 +5,30 @@ import("//tools/grit/grit_rule.gni")
 action("sign_app") {
   script = "//brave/build/android/sign_apk.py"
   _apksigner = "$android_sdk_build_tools/apksigner"
+  _jarsigner = "//third_party/jdk/current/bin/jarsigner"
   _zipalign = "$android_sdk_build_tools/zipalign"
 
   deps = [ "//brave/app/android:create_dist_zips" ]
 
-  target_sign_app_path = ""
-  if (target_cpu == "arm64" || target_cpu == "x64" ||
-      target_apk_base == "mono") {
-    target_sign_app_path = "$root_out_dir/apks/MonochromePublic.apk"
+  if (target_cpu == "arm64" || target_cpu == "x64") {
+    target_sign_app_path = "$root_out_dir/apks/MonochromePublic6432.aab"
   } else {
-    target_sign_app_path = "$root_out_dir/apks/ChromePublic.apk"
+    if (target_apk_base == "mono") {
+      #target_sign_app_path = "$root_out_dir/apks/MonochromePublic.apk"
+      target_sign_app_path = "$root_out_dir/apks/MonochromePublic.aab"
+    } else {
+      #target_sign_app_path = "$root_out_dir/apks/ChromePublic.apk"
+      target_sign_app_path = "$root_out_dir/apks/ChromeModernPublic.aab"
+    }
   }
 
-  outputs = [ "$target_sign_app_path-singed" ]
+  outputs = [ "$target_sign_app_path-signed" ]
```

Comment:
:) good catch @wchen342

Reply:
I fixed this in the past and it broke thinks even though it seems unused.